content/tracing/languages/go.md

```diff
@@ -89,19 +89,46 @@ The Go tracer includes support for the following data stores and libraries.
 
 **Note**: The [integrations documentation][53] provides a detailed overview of the supported packages and their APIs, along with usage examples.
 
+Packages must be imported, i.e.:
+
+```go
+import "gopkg.in/DataDog/dd-trace-go.v1/contrib/<package_dir>/<package_name>"
+```
+
 ## Configuration
 
-The tracer is configured with options parameters when the `Start` function is called. A list of available options are:
+The tracer is configured with options parameters when the `Start` function is called. An example for generating a trace:
 
 ```go
 package main
 
-import "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+import (
+    "net/http"
+    "strings"
+    "log"
+    httptrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/net/http"
```

Comment:
Nits: I actually don't _think_ specifying `httptrace` is needed here, it will be done automatically. cc @Simwar ?

Reply:
depends if you use it as "httptrace" in the code.
In this casse, you need it, from my understanding.